tests/unittests/Foundation/NSPointerFunctionsTests.mm

```diff
@@ -75,7 +75,7 @@ NSUInteger NSPointerFunctionsTestStructSizeFunction(const void* item) {
 TEST(NSPointerFunctions, OpaquePersonality) {
     NSPointerFunctions* functions = [NSPointerFunctions pointerFunctionsWithOptions:NSPointerFunctionsOpaquePersonality];
 
-    const void* ptr1;
+    const void* ptr1 = (void*)0x3;
```

Comment:
The release build was failing here due to new optimizations; it was never valid to use `ptr1` or `ptr3` (derived from `ptr1`) for any reason except, perhaps, addressof.